roles/htcondor/templates/install_condor_poller.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+OLD_CFG=/opt/cloudscheduler/etc/cloudscheduler/condor_poller.yaml
+NEW_CFG=/tmp/cloudscheduler/etc/cloudscheduler/condor_poller.yaml
+
+cd /tmp
+wget --no-check-certificate https://elephant77.heprc.uvic.ca/repo/condor_poller.tar.gz
```

Comment:
URL needs to be replaced

Reply:
I've now deleted this file, I don't think it was ever actually  used in the playbook.